src/components/index.ts

```diff
@@ -23,6 +23,7 @@ export { RadioGroup } from "./RadioGroup/RadioGroup";
 export { SearchField } from "./Input/SearchField";
 export { Select } from "./Select/Select";
 export { SidebarNavigationItem } from "./SidebarNavigationItem/SidebarNavigationItem";
+export { SidebarNavigationTitle } from "./SidebarNavigationItem/SidebarNavigationTitle";
```

Comment:
let's have all of them at top level

`SidebarNavigationItem`
`SidebarNavigationTitle`
`SidebarCollapsibleItem`